### ISSUE TITLE
utillinuxMinimal not in nixpkgs/master and nixos-unstable

### ISSUE THREAD

**Issue by benneti** [NixOS/nix#4649]
**Describe the bug**
the flake.nix relies on utillinuxMinimal that does not exist in the master and nixos-unstable branch anymore (at least I could not find it using rg and the build of nix fails complaining about non defined utillinuxMinimal).  

**Steps To Reproduce**

1. use nixos with flakes
2. upgrade to the latest commit on master or nixos-unstable
3. use the nix flake provided in this repository
4. try to rebuild the system

**Expected behavior**

It would be nice if the flake could be used directly even from nixos-unstable

**`nix-env --version` output**
nix-env (Nix) 2.4pre20210308_1c0e3e4

**Additional context**
a workaround is having a small overlay that defines 
``` nix
final: prev: {
  utillinuxMinimal = prev.util-linux.override {
    minimal = true;
  }; # HACK somthing needs this alias for now
}
```

**Comment by benneti** [comment]
In case this is intentional, see https://github.com/NixOS/nix/pull/4309 and I should not follow nixos-unstable with the nix flake even when running nixos-unstable feel free to close.

**Comment by stephank** [comment]
I believe this is fixed in ce791535f63502215d3d41b6ca8d9e62c5fb72e9

**Comment by benneti** [comment]
can confirm that it works again, thanks for the heads up!
